"""
The functions in this module were created specifically for the purposes
of this project and were not created with error handling in mind, so there
aren't many assertions to detect invalid arguments. 
"""
from . import fn
from . import callback